__tests__/app.test.js

```diff
@@ -95,6 +95,27 @@ describe('5: /api/reviews/:review_id', () => {
                 expect(body.msg).toBe('invalid user input');
             });
     });
+    describe('10: /api/reviews/;review_id (comment count)', () => {
+        test('GET: 200 - review response object includes a comment_count property: total count of all comments with this review_id', () => {
+            return request(app)
+                .get('/api/reviews/2')
+                .expect(200)
+                .then(({body}) => {
+                    expect(body.review).toMatchObject({
+                        review_id: 2,
+                        title: expect.any(String),
+                        review_body: expect.any(String),
+                        designer: expect.any(String),
+                        review_img_url: expect.any(String),
+                        votes: expect.any(Number),
+                        category: expect.any(String),
+                        owner: expect.any(String),
+                        created_at: expect.any(String),
+                        comment_count: '3'
+                    })
+                })
+        });
+    });
```

Comment:
Solid test here! If you have already asserted the properties exist on the review object, this is the whole point we've got you to use `.toMatchObject` this whole time! You can rely on your previous tests responsible for asserting the presence of these properties - just need to now test for the existence of the `comment_count`